test.ts

```diff
@@ -0,0 +1,74 @@
+import { unstable_dev } from "wrangler";
+import type { UnstableDevWorker } from "wrangler";
+
+describe("Worker", () => {
+  let worker: UnstableDevWorker;
+
+  interface AppleAppSiteAssociation {
+    activitycontinuation: {
+      apps: string[];
+    };
+    webcredentials?: {
+      apps: string[];
+    };
+    applinks?: {
+      apps: string[];
+      details: Array<{
+        appID: string;
+        paths: string[];
+      }>;
+    };
+  }
+
+  interface AssetLinks {
+    relation: string[];
+    target: {
+      namespace: string;
+      package_name: string;
+      sha256_cert_fingerprints: string[];
+    };
+  }
+
+  beforeAll(async () => {
+    worker = await unstable_dev("./worker.ts", {
+      experimental: { disableExperimentalWarning: true },
+    });
+  });
+
+  afterAll(async () => {
+    await worker.stop();
+  });
+
+  it("should return the apple-app-site-association JSON on the correct route", async () => {
```

Comment:
Nice @brainbicycle - I couldn't quite figure out how to test that other worker in MP 👌

Reply:
Yeah i thought this was gonna be a pain but was pleasantly surprised! https://developers.cloudflare.com/workers/wrangler/api/

Reply:
Oh I tried to get some basic test going in Metaphysics but was unsuccessful... possibly due to the worker's usage of the `cache` API.